perfrunbook/debug_hw_perf.md

```diff
@@ -138,6 +146,10 @@ Backend stalls are caused when the CPU is unable to make forward progress execut
 6. If back-end stalls due to the cache-system and memory system are the problem, the data-set size and layout needs to be optimized.
 7. Proceed to [Section 6](./optimization_recommendation.md) to view optimization recommendations for working with a large data-set causing backend stalls.
 
+### Drill down Vectorization
+
+Vectorization is accomplished either by SVE or NEON instructions.  SVE vectorization will use 256-bit vectors on Graviton 3 processors, but the scalable nature of SVE makes the binary compatible with other ARM processors with different vector sizes.  NEON vectorization is always a 128-bit vector size, and does not have the predicate feature of SVE.  For SVE instructions there are metrics which describe how many SVE instructions had empty, full and partially-filled SVE predicates.  These metrics apply to loads, stores, integer, and floating-point operations.  There are also 3 counters for floating point operation counters: `flop-scalar-pkc` counts non-vectorized floating point operations.  `flop-fixed-pkc` counts both scalar and NEON floating point operations, accounting for the size of the 128-bit NEON vector (a single-precision NEON op would count 4 times).  `flop-sve-pkc` counts SVE floating point operations, again including the size of the vector (a single-precision SVE operation would count 8 times on a Graviton 3).  The SVE flops are counted assuming full SVE predicate.
```

Comment:
Is this `flop-fixed-pkc` or `flop-mixed-pkc`? I prefer `mixed` because you are running scalar + Neon vector ops. `fixed` has a different connotation in that it may mean `fixed-point` math.

Reply:
I've rewritten this whole paragraph.  Please take a look.